parachain/runtime/litentry/src/weights/pallet_treasury.rs

```diff
@@ -148,4 +103,8 @@ impl<T: frame_system::Config> pallet_treasury::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().writes((3_u64).saturating_mul(p.into())))
 			.saturating_add(Weight::from_parts(0, 5206).saturating_mul(p.into()))
 	}
+	fn spend_local() -> Weight { todo!() }
+	fn payout() -> Weight { todo!() }
+	fn check_status() -> Weight { todo!() }
+	fn void_spend() -> Weight { todo!() }
```

Comment:
Hmm what's wrong with pallet-treasury? 👀

Reply:
pallet_treasury benchmark is skipped for the moment. There is some error within the dependency crate. Spent some time investigate, but didn't succeed. But for the building process, these functions need to be kept here.